3-semantic-html/semantic.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Sematic HTML</title>
+    </head>
+    <body>
+        <header>
+            <h1>Semantic Exercise HTML </h1>
+            <h3>Raúl Valencia</h3>
```

Comment:
Where is the `<h2>` ?. If you prefer you can change the `<h3>` to `<h2>`. It depends on the headers that you will have on the web page